INSTRUCTIONS.md

```diff
@@ -1,45 +1,33 @@
-- if we evaluate otherwise, we'll share our feedback and findings with you
-
-As said, feel free to make some concessions under time pressure. **Please, [document your concessions and decisions](DOCUMENTATION.md) before sharing the assignment with us.**
-
-## Technology and Frameworks
-
-Our application will be built with React Native. It is therefore recommended to use React Native and Expo to get up and running quickly and this repository has Expo set up for you. However if you have another technology in mind you think is better suited, you’re free to use it - as long as we can easily debug and run the application in iOS and Android simulators.
+When searching for an accommodation, FindHotel customers are required to provide the number of guests and rooms for that particular stay. We use the provided information to search for available hotels based on the user's input.
 
-We use Redux at FindHotel, but again you're free to choose any state management technology you'd like.
+Check the following screenshots for the designs to build. You can focus on the Guest and Room Selector, **building the home screen is not required**.
 
-Finally, we prefer to use a typed dialect such as TypeScript or Flow, so it is a plus for you to showcase the use of it, but you are allowed to use any JS flavor.
-
-Styling can be done via CSS, or a CSS-in-JS framework of your choice (we use <https://github.com/emotion-js/emotion>).
-
-## The assignment
+| Home screen                     | Guest and Room Overlay                                                      | Guest and Room Overlay with more rooms                                                                         |
+| ------------------------------- | --------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| ![Home](Home.jpg "Home screen") | ![Guest and Room Overlay](GuestPicker-Default.jpg "Guest and Room Overlay") | ![Guest and Room Overlay with more rooms](GuestPicker-More-Rooms.jpg "Guest and Room Overlay with more rooms") |
 
-Please note for this assignment you are not required to build any server side or API logic, you can build all logic in the application.
+### Time to spend on the assignment ⏳
 
-### The Guest and Room Selector component
+We don't assign a fixed time deadline for it, but after you’ve read the assignment, please let us know when you’re planning to complete it. After you complete the assignment we'll review it internally.
 
-Please see the following screenshots for the designs to build. You can focus on the Guest and Room Selector, **building the home page is not required**.
+- if we evaluate it to be positive, we'll schedule a technical interview to discuss the results together
+- if we evaluate otherwise, we'll share our feedback and findings with you
```

Comment:
I couldn't suggest the complete line, but I would separate the time statement from the evaluation, and put this after the "concessions" statement in the next line.

```suggestion
### Evaluation and next steps

- if we evaluate your assignment positively, we'll schedule a technical interview to discuss the results together,
- otherwise, we'll share our feedback and findings with you
```